### ISSUE TITLE
Fallback to any for `oneOf` with bound identifier

### ISSUE THREAD

**Issue by marionebl** [KnisterPeter/react-to-typescript-definitions#232]
When passing the following source to any of the public methods

```js
import React from 'react';

const fooOrBar = ['foo', 'bar'];

export default class SomeComponent extends React.Component {
   static propTypes = {
     someEnum: React.PropTypes.oneOf(fooOrBar)
   };
}
```

an error is logged, but not thrown:

```
Failed to infer PropType; Fallback to any
TypeError: Cannot read property 'map' of undefined
    at getEnumValues (~/node_modules/react-to-typescript-definitions/dist/src/types.js:105:22)
    at getComplexType (~/node_modules/react-to-typescript-definitions/dist/src/types.js:72:31)
    at Object.get (~/node_modules/react-to-typescript-definitions/dist/src/types.js:16:27)
    at ~/node_modules/react-to-typescript-definitions/dist/src/typings.js:67:30
    at Array.forEach (native)
    at createPropTypeTypings (~/node_modules/react-to-typescript-definitions/dist/src/typings.js:66:9)
    at ~/node_modules/react-to-typescript-definitions/dist/src/typings.js:35:17
    at Array.forEach (native)
    at Object.createTypings (~/node_modules/react-to-typescript-definitions/dist/src/typings.js:26:20)
    at generateFromAst (~/node_modules/react-to-typescript-definitions/dist/src/index.js:56:22)
```

**Comment by marionebl** [comment]
:rocket:

**Comment by KnisterPeter** [comment]
Released as 0.17.0
